zeg/__main__.py

```diff
@@ -181,6 +181,12 @@ def _add_standard_args(parser):
         action='store_true',
         help='Enable verbose logging.',
     )
+    parser.add_argument(
```

Comment:
Can remove this now then

Reply:
Actually, I was assigning to the config object via args and hadn't intended for it to be specified in the yaml, that might be better, I'll raise the PR for mime types and see what you think, it's branched off of this

Reply:
Could just have both?

Reply:
Have made it so you can specify either via a CLI arg or as the configuration object, the configuration yaml takes precedence

Reply:
Possibly. Although i feel it's much more intuitive to have it next to where the paths are defined.

Reply:
Will remove the CLI arguments then